wom/src/main/resources/reference.conf

```diff
@@ -0,0 +1,4 @@
+yaml {
+  # The maximum number of elements that will be parsed in a Yaml
+  max-nodes = 10000
```

Comment:
My personal preference would be to make this much higher than you'd reasonably expect a standalone user to hit by default. IMO 10,000 is not our of the realms of possibility

We can always make it "safer than the default" in our hosted instances.